web/apps/apps.go

```diff
@@ -214,6 +257,36 @@ func installHandler(installerType consts.AppType) echo.HandlerFunc {
 	}
 }
 
+// logsHandler handles all POST /:slug/logs requests and forwards to sent log
+// lines to the server logger.
+func logsHandler(appType consts.AppType) echo.HandlerFunc {
+	return func(c echo.Context) error {
+		inst := middlewares.GetInstance(c)
+		slug := c.Param("slug")
+		if err := middlewares.AllowWholeType(c, permission.POST, consts.AppLogs); err != nil {
+			return err
+		}
```

Comment:
I would also check that the token comes from the flagship app (`middlewares.AllowMaximal`) or that the slug is the same as the one of the token app (`middlewares.AllowFromKonnector`). I don't want that konnector `foo` creates logs for konnector `bar` by mistake (or malicious behavior).